docs/learn/learn-comparisons.md

```diff
@@ -1,63 +1,196 @@
+| **Scalability**              | High scalability, constrained by computational complexity of zk-SNARKs or zk-STARKs.                           | High, with parallelization, but limited by gas costs on L1 chains like Ethereum.                                                                                             | Inherently scalable through native execution sharding and parachains operating in parallel. [Pipelining](./learn-async-backing.md) and [core scheduling](./learn-agile-coretime.md) increase throughput and scalability for the single rollup. Execution sharding is enabled by multiple virtual cores using [coretime](./learn-agile-coretime.md). |
+| **Decentralization**         | Centralization risks for sequencers and zk proof computation.                                                  | Sequencer centralization risks; fewer validators can lead to potential censorship.                                                                                           | Collator decentralization is critical to ensure parachains remain trustless and independent.                                                                                                                                                                                                                                                        |
+| **Interoperability**         | Limited interoperability, often restricted to compatible L1s.                                                  | Limited interoperability, often confined to the parent blockchain ecosystem. See the [comparison about interoperability](#interoperability-comparison) for more information. | Native interoperability through [XCM](./learn-xcm.md), allowing seamless communication between parachains having different logic. [Trustless bridges](./learn-bridges.md) can connect Polkadot to other blockchains.                                                                                                                                |
+| **State Transition Logic**   | General-purpose but constrained by zk-circuit implementation complexity.                                       | Supports arbitrary state transitions with assumptions of optimistic validity.                                                                                                | Each parachain can define its unique state transition function (STF), compiled to Wasm and validated per Parachain Protocol rules.                                                                                                                                                                                                                  |
```

Comment:
Optimistic rollups can only support evm state transitions since they have to be checked in the ecm